src/main/java/com/github/demo/model/Book.java

```diff
@@ -53,4 +57,21 @@ public String getCover() {
     public void setCover(String cover) {
         this.cover = cover;
     }
+
+    public long getRating() {
+        return rating;
+    }
+
+    public void setRating(long rating) {
+
+        String tempRating = "";
+
+        if (rating < 0) {
+            this.rating = 0;
+        } else if (rating > 5) {
+            this.rating = 5;
+        } else {
+            this.rating = 0;
```

Comment:
Hey, I don't think this should be `0`, maybe you could do this:
```suggestion
            this.rating = rating;
```